.github/workflows/ci.yaml

```diff
@@ -77,85 +82,115 @@ jobs:
       matrix:
         CNI: ["default", "calico"]
     steps:
-      - name: Set up /bin permissions   # 1. Set up /bin permissions
+      - name: Set up /bin permissions
```

Comment:
Why are we setting /bin world writable?

Reply:
It requires for some donwloaeded tools. We already have this in main version of ci files.